lib/order.rb

```diff
@@ -7,12 +9,66 @@ def initialize(id, products)
       @products = products
     end
 
+    def self.all
+      all_orders = []
+      CSV.open("support/orders.csv", "r").each do |line|
+        id = line[0]
+        product = line[1]
+        instance_order = Order.new(id, string_product_to_hash(product))
+        all_orders << instance_order
+      end
+      return all_orders
+    end
+
+    def self.string_product_to_hash(string_product)
+      #Slivered Almonds:22.88;Wholewheat flour:1.93;Grape Seed Oil:74.9
+      hash_products = {}
+      result = string_product.split(";")
+      result.each do |i|#Slivered Almonds:22.88
+        name_and_price = i.split(":")
+        name = name_and_price[0]
```

Comment:
I like this comment - it does a good job of informing the reader what the data ought to look like at this point.